pkgs/desktops/gnome/extensions/extensionOverrides.nix

```diff
@@ -47,9 +48,11 @@ super: lib.trivial.pipe super [
 
   (patchExtension "ddterm@amezin.github.com" (old: {
     nativeBuildInputs = [ gobject-introspection wrapGAppsHook ];
-    buildInputs = [ vte ];
+    buildInputs = [ vte libhandy ];
+    postPatch = ''
+      substituteInPlace "bin/com.github.amezin.ddterm" --replace-fail "/usr/bin/gjs" "${gjs}/bin/gjs"
+    '';
```

Comment:
`> substituteStream(): ERROR: pattern /usr/bin/gjs doesn't match anything in file 'bin/com.github.amezin.ddterm'` when this pr is rebased on master